Katello/katello
# [WIP] Add bastion to webpack

This commit adds bastion to webpack. This will allow katello's angular pages to be moved over to webpack.

The content credentials page is moved as well to ensure bastion is working.

to test this out:
- npm install
- Add this change to Foreman:
```diff
diff --git a/config/webpack.config.js b/config/webpack.config.js
index 2bb234f..c166f57 100644
--- a/config/webpack.config.js
+++ b/config/webpack.config.js
@@ -164,6 +164,10 @@ module.exports = env => {
             fallback: 'style-loader', // The backup style loader
             use: production ? 'css-loader!sass-loader' : 'css-loader?sourceMap!sass-loader?sourceMap'
           })
+        },
+        {
+          test: /\.html$/,
+          loader: 'raw-loader'
         }
       ]
     },

```
- restart server
- Navigate to `/content_credentials2`

TODO:

- [x] Move the rest of bastion to webpack
- [x] Fix linting
- [ ] Fix testing
- [ ] Ensure all bastion dependencies are present
- [ ] Move the rest of content credentials page and ensure functionality
- [ ] Fix translations and ensure they are working
- [ ] Check katello plugins are still working: rubygem-foreman_scc_manager,rubygem-redhat_access,rubygem-foreman_virt_who_configure
